package-json-script-names.md

```diff
@@ -0,0 +1,28 @@
+# `package.json` Script Names
+
+## General
+
+- `npm run validate`: Runs all tests, type checks, linters, and formatters
+
+## Testing
+
+- `npm run test`: Runs all test runners (Jest and/or Cypress)
+- `npm run test:watch`: Runs all test runners in watch mode
+- `npm run test:jest`: Runs Jest and exits
+- `npm run test:jest:watch`: Runs Jest in watch mode
+- `npm run test:cypress`: Runs Cypress and exits
+- `npm run test:cypress:open`: Open Cypress dashboard
+
+## Linting
+
+- `npm run lint`: Runs all linters and formatters, and fixes any auto-fixable problems
```

Comment:
Worth mentioning that this setup assumes we're running Prettier as part of the ESLint and Stylelint setups? And if you're on a project where you want to keep prettier separate, it'll be `npm run lint:prettier`?

Reply:
@spaceninja I was trying to be more general by saying "linters and formatters" rather than "eslint and prettier"